pulsar-client-schema/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -104,6 +105,17 @@ default void validate(byte[] message) {
         return JSONSchema.of(clazz);
     }
 
+    /**
+     * Key Value Schema whose underneath schemas are AvroSchema.
+     */
+    static Schema<?> KeyValue(Class key, Class value) {
```

Comment:
actually I think it might be better to do following:

- be able to construct keyvalue schema with `SchemaType`
- make the default as JSON, since we are using JSON as default schema for other places.

```
static <K, V> Schema<KeyValue<K, V>> KeyValue(Class<K> key, Class<V> value, SchemaType type) {
   checkArgument(SchemaType.JSON == type || SchemaType.AVRO == type);
   ...
} 

static <K, V> Schema<KeyValue<K, V>> KeyValue(Class<K> key, Class<V> value) {
   return KeyValue(key, value, SchemaType.JSON);
   ...
}
```